src/util/conversions.rs

```diff
@@ -70,6 +78,7 @@ pub fn bytes_to_pages_up(bytes: usize) -> usize {
     raw_align_up(bytes, BYTES_IN_PAGE) >> LOG_BYTES_IN_PAGE
 }
 
+/// Convert size in bytes to the number of pages (aligned down)
```

Comment:
The doc comment says the result is aligned *down*, but it calls `bytes_to_pages_up`.  Is this a mistake?

Reply:
Yes, it was a mistake when I added the comment. I thought it was 'align down', as we have the other function for aligning up. 

I think it is a bit confusing to have two functions for the same thing but with different names. I removed this function, and use `bytes_to_pages_up` instead.